cmd/uhc/main.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/openshift-online/uhc-cli/cmd/uhc/cluster"
 	"github.com/openshift-online/uhc-cli/cmd/uhc/completion"
+	"github.com/openshift-online/uhc-cli/cmd/uhc/config"
```

Comment:
You are importing `config` but the name of the package is `configcmd`. Either change the name of the package or change this import.